build_helpers/buildAPIDocs.sh

```diff
@@ -31,7 +31,7 @@ var FILES_TO_READ = [
   },
   {
     path: path.join(PROJECT_ROOT, 'src/FixedDataTableCellDefault.js'),
-    name: 'Cell',
+    name: 'DataCell',
```

Comment:
`DataCell` is our new default Cell component.
The previous `Cell` component has been deprecated.